docs/watcher/README.md

```diff
@@ -57,3 +57,9 @@ metadata:
     results.tekton.dev/recordSummaryAnnotations: |-
       {"foo": "bar"}
 ```
+
+## Resource Deletion
+
+When the command line flag is `completed_run_grace_period` is set to any value other than `0`, resources will be deleted on completion. If < `0`, Runs will be deleted immediately.
```

Comment:
That sounds a bit confusing to me. If the value is set to 100, wouldn't the runs be deleted after that grace period, instead of "will be deleted on completion"?

Reply:
Yes, I'll change the description here.

Reply:
Changed.